wlauto/common/android/device.py

```diff
@@ -670,6 +670,18 @@ def get_device_model(self):
         except KeyError:
             return None
 
+    def is_network_connected(self, network_host='www.google.com'):
```

Comment:
Just spotted -- the method body is over-indented; each indentation level must be exactly four spaces; have you run the pep8/pylint checkers on this?